Sources/BrowserServicesKit/SecureVault/SecureVaultModels.swift

```diff
@@ -489,7 +489,7 @@ extension SecureVaultModels.CreditCard: SecureVaultAutofillEquatable {
 // MARK: - WebsiteAccount Array extensions
 extension Array where Element == SecureVaultModels.WebsiteAccount {
 
-    public func sortedForDomain(_ targetDomain: String, tld: TLD, removeDuplicates: Bool = false) -> [SecureVaultModels.WebsiteAccount] {
+    public func sortedForDomain(_ targetDomain: String, tld: TLD, removeDuplicates: Bool = false, alphabeticalOnly: Bool = false) -> [SecureVaultModels.WebsiteAccount] {
```

Comment:
@afterxleep I keep observing the issue in the macOS app. Did you forget to use `alphabeticalOnly: true` where needed?